packages/sdk/src/HopBridge.ts

```diff
@@ -520,7 +529,12 @@ class HopBridge extends Base {
         destinationTxFee,
         destinationChain
       )
+    }
 
+    let totalFee
+    if (isSourceL1) {
+      totalFee = BigNumber.from(0)
```

Comment:
I think these duplicated conditional checks can be combined

```ts
let adjustedBonderFee
let adjustedDestinationTxFee
let totalFee
if (isSourceL1) {
  // there are no Hop fees when the source is L1
  adjustedBonderFee = BigNumber.from(0)
  adjustedDestinationTxFee = BigNumber.from(0)
  totalFee = BigNumber.from(0)
} else {
  if (isHTokenSend) {
    // fees do not need to be adjusted for AMM slippage when sending hTokens
    adjustedBonderFee = bonderFeeRelative
    adjustedDestinationTxFee = destinationTxFee
  } else {
    // adjusted fee is the fee in the canonical token after adjusting for the hToken price
    adjustedBonderFee = await this.calcFromHTokenAmount(
        bonderFeeRelative,
        destinationChain
      )

    adjustedDestinationTxFee = await this.calcFromHTokenAmount(
          destinationTxFee,
          destinationChain
     )
  }

  // enforce bonderFeeAbsolute after adjustment
  const bonderFeeAbsolute = await this.getBonderFeeAbsolute()
  adjustedBonderFee = adjustedBonderFee.gt(bonderFeeAbsolute)
    ? adjustedBonderFee
    : bonderFeeAbsolute
  totalFee = adjustedBonderFee.add(adjustedDestinationTxFee)
}

```

Reply:
Good call. Resolved with addb6908e0a46f745f05afd98071c01726e72846.